packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -192,6 +196,65 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		assert(ackNackResult.success, "summary op should be acked");
 	});
 
+	it("sample use for customized containerRuntimeWithBlob", async () => {
```

Comment:
This should be a unit test in container runtime. There is nothing end-to-end about it :-).